service/models.py

```diff
@@ -15,6 +15,7 @@
 restock_level (number) - the restock level of the Inventory item
 
 """
+# pylint: disable=import-error, superfluous-parens
```

Comment:
I noticed no modifications in `models.py`. May I ask why the pylint disabling was added in this file?

Reply:
The build failed without it.